Header header
int32 state
int32 next_state

robot_msgs/Twist viz_offset
robot_msgs/Twist viz_error